### DISCUSSION TITLE
Motion detection and saving *.mp4

### DISCUSSION THREAD

**Discussion by ArclonT** [roleoroleo/yi-hack-Allwinner-v2#171] [Q&A]
Hi,
how to disable motion detection and saving mp4 files?
Camera is doing it. MQTT is disabled.

**Comment by roleoroleo** [comment]
Is cloud enabled?

**Reply by ArclonT** [reply]
No. I think not.

**Reply by roleoroleo** [reply]
Disable this check:
![immagine](https://user-images.githubusercontent.com/39277388/119485218-1eb64c00-bd57-11eb-93f5-4b2bcf614c4a.png)

**Comment by ArclonT** [comment]
Still recording :(
![image](https://user-images.githubusercontent.com/61750494/119493791-dbf97180-bd60-11eb-8be8-81d76fa5bb3a.png)

**Comment by marcomehnert** [comment]
Solution that worked for me: https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/17#issuecomment-770238074

**Reply by roleoroleo** [reply]
I will try to fix it.

**Comment by roleoroleo** [comment]
I don't understand this behavior because when cloud is disabled watch_process is not running.
So, why mp4record is restarted?

**Comment by adolfocartagena** [comment]
Is there any script that launches the recording for a certain amount of time (let's say one minute)? I want to control the detection from external software but have it saved to the SD card.
Thanks

**Reply by roleoroleo** [reply]
Yes.
`ipc_cmd -S XX`
